samples/net/gsm_modem/prj.conf

```diff
@@ -0,0 +1,34 @@
+
+# UART support
+CONFIG_SERIAL=y
+
+# GSM modem support
+CONFIG_MODEM=y
+CONFIG_MODEM_GSM_PPP=y
+CONFIG_MODEM_GSM_UART_NAME="UART_1"
+
+# PPP networking support
+CONFIG_NET_PPP=y
+CONFIG_NET_L2_PPP=y
+CONFIG_NET_NATIVE=y
+CONFIG_NETWORKING=y
+
+CONFIG_NET_L2_PPP_TIMEOUT=10000
+
+# IPv4 enables PPP IPCP support
+CONFIG_NET_IPV4=y
+CONFIG_NET_IPV6=n
+
+# Network management events
+CONFIG_NET_CONNECTION_MANAGER=y
+
+# Log buffers, modem and PPP
+CONFIG_LOG_BUFFER_SIZE=16384
```

Comment:
We need to modify the log settings here. I suggest that this is changed to (by default the debug logging should not be enabled). Also \n needs to be added to the end of the file.
```
CONFIG_LOG=y
CONFIG_NET_LOG=y
#CONFIG_LOG_BUFFER_SIZE=16384
#CONFIG_LOG_STRDUP_BUF_COUNT=200
#CONFIG_MODEM_LOG_LEVEL_DBG=y
#CONFIG_NET_PPP_LOG_LEVEL_DBG=y
#CONFIG_NET_L2_PPP_LOG_LEVEL_DBG=y
#CONFIG_NET_MGMT_EVENT_LOG_LEVEL_DBG=y
#CONFIG_NET_CONNECTION_MANAGER_LOG_LEVEL_DBG=y 
```

Reply:
Done